open-codegen/opengen/builder/optimizer_builder.py

```diff
@@ -142,23 +141,22 @@ def __copy_icasadi_to_target(self):
                         ignore=shutil.ignore_patterns(
                             '*.lock', 'ci*', 'target', 'auto*'))
 
-    def __generate_icasadi_header(self):
-        """Generates the header of icasadi, with constant definitions
+    def __generate_icasadi_lib(self):
```

Comment:
Since this function generates the CasADi Rust library, we can remove `opengen/icasadi/src/lib.rs`.

Reply:
I will remove it